pkg/virt-handler/heartbeat/heartbeat.go

```diff
@@ -35,13 +35,15 @@ type HeartBeat struct {
 }
 
 func NewHeartBeat(clientset k8scli.CoreV1Interface, deviceManager device_manager.DeviceControllerInterface, clusterConfig *virtconfig.ClusterConfig, host string) *HeartBeat {
+	const CPUManagerOS3Path = virtutil.HostRootMount + "var/lib/origin/openshift.local.volumes/cpu_manager_state"
+	const CPUManagerPath = virtutil.KubeletRoot + "/cpu_manager_state"
```

Comment:
nit: can you start these with a non-capital letter?
```suggestion
	const cpuManagerOS3Path = virtutil.HostRootMount + "var/lib/origin/openshift.local.volumes/cpu_manager_state"
	const cpuManagerPath = virtutil.KubeletRoot + "/cpu_manager_state"
```